### PR TITLE
Improved overall style constuction & improved file structure

### PR DESCRIPTION
**Pull request by Markvanderuit**
- Lots of small changes and improvements to GUI, such as
  - Global color variables
  - Drawer styling redone and improved upon
  - More closely follows google's guidelines
- Styled two more views
  - Shotcaller view
  - Upload view
- EJS file content moved to element files. This has a few implications:
  - Everything except style.css is dumped overboard. Can't use variables there, so don't use it.
  - Don't add content to EJS files, as I'll just move it out anyways.
- Created some subfolders in /Elements/ because it was a mess.

### CODE REVIEW

File: elements/gotofail-drawer.html [hunk 1]
```diff
@@ -72,6 +74,10 @@
                 background-color: transparent;
             }
 
+            .ripple-style {
```
Comment (martijnjanssen) [inline elements/gotofail-drawer.html]: You can change this to `paper-ripple` and remove the added classes on the `paper-ripple` elements.

File: elements/gotofail-drawer.html [hunk 2]
```diff
@@ -86,37 +92,37 @@
...
                     <iron-icon class="icon-style" icon="image:timer"></iron-icon>
                     <span class="text-style" style="padding-left: 10px;">Shot calling view</span>
                 </paper-item> </a>
 
-                <a href="/director" tabindex="-1" style="text-decoration:none"> <paper-item class="item-style">
-                    <paper-ripple style="color: #3F51B5;"></paper-ripple>
-                    <iron-icon class="icon-style" icon="image:camera-alt"></iron-icon>
-                    <span class="text-style" style="padding-left: 10px;">Director view</span>
+                <a tabindex="-1" style="text-decoration:none"> <paper-item class="item-style">
+                    <!--<paper-ripple class="ripple-style"></paper-ripple>-->
```
Comment (martijnjanssen) [inline elements/gotofail-drawer.html]: Commented code can be removed.

File: elements/gotofail-drawer.html [hunk 3]
```diff
@@ -86,37 +92,37 @@
...
+                    <iron-icon class="icon-style" style="color: rgb(200,200,200); opacity: 0.6;" icon="image:camera-alt"></iron-icon>
+                    <span class="text-style" style="padding-left: 10px; color: rgb(200,200,200);">Director view</span>
```
Comment (martijnjanssen) [inline elements/gotofail-drawer.html]: Style in the element? Should not be needed.

File: elements/gotofail-drawer.html (hunk 1 above)
Comment (Markvanderuit) [inline elements/gotofail-drawer.html]: Changed. Good catch, don't know how that got that way.

**Comment by bart-de-jonge** [comment]
Can't find any weird things, great work!

File: elements/gotofail-drawer.html (hunk 3 above)
Comment (Markvanderuit) [inline elements/gotofail-drawer.html]: Removed to text-style class.

File: public/stylesheets/style.css [hunk 4]
```diff
@@ -10,4 +10,24 @@ header {
 a {
   color: #00B7FF; }
 
+* {
+  margin: 0;
+  padding: 0; }
```
Comment (martijnjanssen) [inline public/stylesheets/style.css:15]: Is the wildcard style really needed? Wouldn't this introduce unexpected behavior?

Comment (Markvanderuit) [inline public/stylesheets/style.css:15]: Was there from the start, actually, just moved up one level in css files. Actually solves some issues with polymer right now. I'd rather leave it in, unless you can show me unexpected behavior. :)

File: elements/user-view.html [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+<link rel="import" href="user-view/gotofail-login.html">
+<link rel="import" href="../public/colors.html">
+
+<dom-module id="user-view">
+    <template>
+        <style>
+
+            paper-toolbar {
+                background-color: var(--main-color-500);
+                color: white !important;
```
Comment (martijnjanssen) [inline elements/user-view.html]: Please avoid the usage of `!important`. Is this really needed?

Comment (Markvanderuit) [inline elements/user-view.html]: Yes, overrides Polymer.

**Comment by martijnjanssen** [comment]
Some small issues, looks good to me.

File: elements/user-view.html (hunk 5 above)
Comment (Markvanderuit) [inline elements/user-view.html]: Nevermind, not in this case. Removed.

File: public/stylesheets/style.css (hunk 4 above)
Comment (bart-de-jonge) [inline public/stylesheets/style.css:15]: wildcard like this is overriden by everything else, so probly no unexpected behavoiur

**Comment by martijnjanssen** [comment]
Thanks!

**Pull request merged**